runtime/lua/vim/secure.lua

```diff
@@ -171,9 +171,7 @@ function M.trust(opts)
   local trust = read_trust()
 
   if action == 'allow' then
-    if not bufnr then
-      return false, 'allow operates on buffers - bufnr is required'
-    end
+    assert(bufnr, 'bufnr is required when action is "allow")
```

Comment:
Looks like you're missing a closing `'`.

```suggestion
    assert(bufnr, 'bufnr is required when action is "allow"')
```